vault-handler/replicas-1/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: vault
+
+resources:
+  - job.yaml
+  - rbac.yaml
+
+images:
+  - name: kubefirst/vault-handler
+    newName: ghcr.io/konstructio/vault-handler
+    newTag: 'v1.0.0'
```

Comment:
I have to wonder if we can remove that and instead point to the actual image overall

Reply:
Done Señor!